packages/contracts-bedrock/snapshots/abi/OptimismPortal.json

```diff
@@ -509,6 +509,19 @@
     "stateMutability": "view",
     "type": "function"
   },
+  {
+    "inputs": [],
+    "name": "spacer_53_0_1",
```

Comment:
This doesn't seem right, I think you need to rebase

Reply:
fixed.